backend/test/institution_request_collection_handler_test.py

```diff
@@ -79,17 +80,11 @@ def test_post(self, verify_token):
 def initModels(cls):
     """Init the models."""
     # Other user
-    cls.other_user = User()
-    cls.other_user.name = 'Other User'
-    cls.other_user.email = ['otheruser@test.com']
-    cls.other_user.put()
+    cls.other_user = mocks.create_user()
```

Comment:
Remove initModels() function and move the creation of the object mocks to the specific test.